GPU/Common/SplineCommon.cpp

```diff
@@ -926,16 +926,19 @@ void DrawEngineCommon::SubmitSpline(const void *control_points, const void *indi
 	patch.patchFacing = patchFacing;
 
 	if (g_Config.bHardwareTessellation && g_Config.bHardwareTransform && !g_Config.bSoftwareRendering) {
-		float *pos = (float*)(decoded + 65536 * 18); // Size 3 float
-		float *tex = pos + count_u * count_v * 3; // Size 3 float
-		float *col = tex + count_u * count_v * 3; // Size 4 float
+		int stride = g_Config.iGPUBackend == GPU_BACKEND_VULKAN || g_Config.iGPUBackend == GPU_BACKEND_DIRECT3D11 ? 4 : 3;
```

Comment:
I find checking for iGPUBackend here a little icky, but I'm not really sure what a better alternative would be ... passing in stride as an argument, or maybe switching to 4 everywhere even though it might waste some space in some configuration?

Anyway, I'd prefer if you put parentheses around the big expression with the checks - I don't trust precedence with ?/:, it's so easy to get wrong.